modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -352,12 +377,18 @@
 	apply_overlay(BODYPARTS_LAYER)
 	update_damage_overlays()
 
-/obj/item/proc/wear_species_version(file_to_use, state_to_use, layer, species)
+/obj/item/proc/wear_species_version(file_to_use, state_to_use, layer, species, default_file_to_use)
 	return
 
-/obj/item/clothing/wear_species_version(file_to_use, state_to_use, layer, species)
+//generates a species specific icon for a state_to_use icon in file_to_use if one doesnt exist
+//said species specific icon will be generated using default_file_to_use
+//layer specifies the sprite layer the sprite will be on
+//species is for the specific species the icon will be generated for
+//
+//also fuck vsc, DREAM MAKER FOR LIFE!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
That's not how documentation works, it's `///`. Plus, if you're doing more than one line, you would do something like this:
```suggestion
/**
 * Generates a species specific icon for a state_to_use icon in file_to_use if one doesnt exist
 * said species specific icon will be generated using default_file_to_use
 * layer specifies the sprite layer the sprite will be on
 * species is for the specific species the icon will be generated for
 */
```

Reply:
That's still also the wrong format properly speaking:
```byond
/**
 * Generates a species-specific clothing icon.
 * 
 * Arguments:
 * * file_to_use - Icon file to use for clothing sprite
 * * state_to_use - Icon state to use within file_to_use
 * * layer - specifies the sprite layer the sprite will be on
 * * species - the specific species the icon will be generated for
 * * default_file_to_use - default fallback icon to use
 */
```